stories/example-screen-form-view.stories.js

```diff
@@ -0,0 +1,451 @@
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="collapse" id="collapseInfoThree">
+      <div class="card-body text-left pt-0 fs-6">
+        <div class="row mb-2">
+          <div class="col-md-4 col-sm-12">Charges</div>
+          <div class="col-md-8 col-sm-12 text-muted">Lawrence Herrera</div>
+        </div>
+      </div>
+    </div>
+  </div>
+`
+const additionalInfoFour = `
+  <div class="card bg-light border-0 mb-2">
+    <div class="card-header bg-light mb-0 rounded">
+      <div class="d-flex justify-content-between">
+        <button class="d-flex text-left btn text-warning text-decoration-none p-0 mb-0" data-bs-toggle="collapse" href="#collapseInfoFour" aria-expanded="false" aria-controls="collapseInfoFour">
+          <span class="tabler-chevron-down lh-1 my-0 me-1">
+            <svg width="1rem" height="1rem">
+              <use xlink:href="${iconSprite}#tabler-chevron-down" />
+            </svg>
+          </span>
+          <span class="fs-5 fw-bolder lh-sm">
+            Meetings
+          </span>
+        </button>
+      </div>
+    </div>
+    <div class="collapse show" id="collapseInfoFour">
+      <div class="card-body text-left pt-0 fs-6">
+        <div class="d-flex justify-content-between mb-2">
+          <svg width="1.125rem" height="1.125rem" class="mt-1 me-2" role="button" tabindex="0">
+            <use xlink:href="${iconSprite}#tabler-file-search" />
+          </svg>
+          <div class="d-flex flex-column w-100">
+            <div>
+              <span class="me-1">Date:</span>
+              <span class="text-muted">2019-06-14 12:03:00</span>
+            </div>
+            <div>
+              <span class="me-1">Status:</span>
+              <span class="text-muted">Meeting Scheduled</span>
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+        <div class="d-flex justify-content-between mb-2">
+          <svg width="1.125rem" height="1.125rem" class="mt-1 me-2" role="button" tabindex="0">
+            <use xlink:href="${iconSprite}#tabler-file-search" />
+          </svg>
+          <div class="d-flex flex-column w-100">
+            <div>
+              <span class="me-1">Date:</span>
+              <span class="text-muted">2019-06-14 12:03:00</span>
+            </div>
+            <div>
+              <span class="me-1">Status:</span>
+              <span class="text-muted">Meeting Scheduled</span>
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+`
+
+const tabsNavigationHtml = `
+  <div class="w-100 overflow-auto p-0 mb-0 border border-light">
+    <div class="bg-white rounded-top border border-light d-flex">
```

Comment:
This is causing a double-border on the navbar, let's change to this:

```
<div class="w-100 overflow-auto p-0 mb-0 border-light border-bottom">
    <div class="bg-white rounded-top d-flex">
```
broken:
![2021-09-08_08-28-32AM](https://user-images.githubusercontent.com/1605754/132518421-175048b8-fd5d-4e6c-b2bb-5af9e78ebd6d.png)

fixed:
![2021-09-08_08-29-39AM](https://user-images.githubusercontent.com/1605754/132518633-157080ca-487e-4b61-a764-f56287cf214f.png)